code/src/index.css

```diff
@@ -1,13 +1,272 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+	  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+	  sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+  }
+ 
+  /* movie-page */
+
+.all-movies-container {
+	background-color: black;
+	display: grid;
+	grid-template-columns: repeat(4, 1fr);
+	font-family: 'Bebas Neue', cursive;
+	font-size: x-large;
+}
+
+.movie-container {
+	position: relative;
+}
+
+img {
+	width: 100%;
+}
+
+.movie-container:hover > img {
+	opacity: 0.5;
+}
+
+.details-overlay {
+	opacity: 0;
+	position: absolute;
+	color: white;
+	bottom: 0;
+	left: 0;
+	width: 70%;
+	padding: 20px;
+}
+
+.movie-container:hover > .details-overlay {
+	opacity: 1;
+}
+
+/* movie details */
+
+.back-link {
+	position: absolute;
+	display: inline-flex;
+	color: #fff;
+	align-items: center;
+	font-weight: 700;
+	text-decoration: none;
+	left: 50px;
+	top: 50px;
+	font-family: 'Bebas Neue', cursive;
+	display: inline-block;
+	background: #e50914;
+	color: #fff;
+	padding: 0.4rem 1.3rem;
+	font-size: 1rem;
+	text-align: center;
+	border: none;
+	cursor: pointer;
+	margin-right: 0.5rem;
+	outline: none;
+	box-shadow: 0 1px 0 rgba(0,0,0,0.45);
+	border-radius: 2px;
+}
+
+.poster {
+	width: 20%;
+	border: solid 5px #fff;
+}
+
+.summary-section {
+	display: flex;
+	align-items: flex-end;
+	padding: 50px;
+	font-family: 'Cairo', sans-serif;
+    font-family: 'Zen Antique Soft', serif;
+}
+
+.details {
+	margin-left: 20px;
+	display: block;
+	box-sizing: border-box;
+}
+
+.movie-title {
+	box-sizing: border-box;
+	display: block;
+	margin-inline-start: 0px;
+	margin-inline-end: 0px;
+	font-weight: bold;
+	color: #fff;
+}
+
+.ratings {
+	box-sizing: border-box;
+	font-size: 17px;
+	color: red;
+	font-weight: 400;
+	margin-left: 10px;
+}
+
+.movie-description {
+	max-width: 400px;
+	box-sizing: border-box;
+	margin-block-start: 1em;
+	margin-block-end: 1em;
+	margin-inline-start: 0px;
+	margin-inline-end: 0px;
+	color: #fff;
+}
+
+.showcase {
+	display: flex;
+	justify-content: center;
+    box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+    --primary-color: #e50914;
+	font-family: 'Arial', sans-serif;
+	-webkit-font-smoothing: antialiased;
+	background: #000;
+	color: #999;
+	-moz-osx-font-smoothing: grayscale;
+	color: #fff;
+	width: 100%;
+    height: 93vh;
+    position: relative;
+    background: url('./asset/wallpaper.jpg') no-repeat center/cover;
+}
```

Comment:
This styling looks so professional! The font, colors and the background.. great work! Never heard of font-smoothing before, thanks! Learned something new :D